themes/jb/layouts/sponsors/list.html

```diff
@@ -0,0 +1,38 @@
+{{ define "main" }}
+    <section class="spnosor-list px-4 mb-5">
+        <div class="container">
+            <div class="column is-full content">
+                <h1 class="title">Sponsors</h1>
+                <h4>We have many sponsors we love who support each show, and we hope you'll enjoy them too:</h4>
+                {{ if .Params.Sponsors -}}
```

Comment:
Since, we're reusing the filtered pages more than once, if we pull it out to a variable we can save on filtering all the pages for the site.

```suggestion
                {{- $sponsor_pages := where site.Pages "Section" "sponsors" -}}
                {{ if .Params.Sponsors -}}
```